cmd/podman/volumes/export.go

```diff
@@ -48,8 +49,13 @@ func export(cmd *cobra.Command, args []string) error {
 	containerEngine := registry.ContainerEngine()
 	ctx := context.Background()
 
-	if targetPath == "" && cmd.Flag("output").Changed {
-		return errors.New("must provide valid path for file to write to")
+	if targetPath == "" {
+		if cmd.Flag("output").Changed {
+			return errors.New("must provide valid path for file to write to")
+		}
+		if term.IsTerminal(int(os.Stdout.Fd())) {
+			return errors.New("refusing to export to terminal. Use -o flag or redirect")
```

Comment:
For consistency it would be good to make the message similar to what load has:
```
cannot read from terminal, use command-line redirection or the --input flag
```
So something like
```
cannot write to terminal, use command-line redirection or the --output flag
```